app/assets/stylesheets/smart_answers.scss

```diff
@@ -4,6 +4,28 @@ $is-ie: false !default;
 // The more generic SmartAnswer CSS is in Static _multi_step.scss
 
 .smart_answer {
+  @include govuk-media-query($media-type: print) {
```

Comment:
We should create a print CSS for the future - it will help keep things separated. 

```
<link rel="stylesheet" media="print" href="print.css">
```

Reply:
Good idea! - turns out there is already a print.scss, so I've move the styles into that file.

Reply:
nice!